debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -96,11 +100,14 @@ public TableSchema create(String schemaPrefix, String envelopSchemaName, Table t
         SchemaBuilder valSchemaBuilder = SchemaBuilder.struct().name(schemaNameAdjuster.adjust(schemaNamePrefix + ".Value"));
         SchemaBuilder keySchemaBuilder = SchemaBuilder.struct().name(schemaNameAdjuster.adjust(schemaNamePrefix + ".Key"));
         AtomicBoolean hasPrimaryKey = new AtomicBoolean(false);
+        Set<Column> pkColumns = new HashSet<Column>(table.primaryKeyColumns());
```

Comment:
I don't think a set can be used here, as the order of PK columns needs to be maintained.

Reply:
Should a LinkedHashSet be enough ?